daemon/chaintopology.c

```diff
@@ -173,6 +173,9 @@ static void connect_block(struct lightningd_state *dstate,
 			add_tx_to_block(b, &txid);
 	}
 	b->full_txs = tal_free(b->full_txs);
+
+	/* Tell peers about new block. */
+	peers_new_block(dstate, b->height);
```

Comment:
We might want to generalize this to a list of callback that are registered, since there are quite a number of places we'd potentially need to be notified when a new block comes in. Is there a nice publish/subscriber implementation somewhere?